docs/book/application-integration/usage-in-a-laminas-mvc-application.md

```diff
@@ -0,0 +1,198 @@
+                ],
+                'validators' => [
+                    [
+                        'name'    => StringLength::class,
+                        'options' => [
+                            'min' => 1,
+                            'max' => 100,
+                        ],
+                    ],
+                ],
+            ],
+            // …
+        ];
+    }
+}
+```
+
+## Using Form
+
+### Create Controller
+
+Using the form in a controller, e.g.
+`module/Album/Controller/AlbumController.php`:
+
+```php
+namespace Album\Controller;
+
+use Album\Form\AlbumForm;
+use Laminas\Form\FormInterface;
+use Laminas\Mvc\Controller\AbstractActionController;
+
+class AlbumController extends AbstractActionController
+{
+    /** @var FormInterface */
+    private $form;
+    
+    public function __construct(AlbumForm $form)
+    {
+        $this->form = $form;
+    }
+    
+    public function addAction()
+    {
+        // Set action attribute
+        $this->form->setAttribute(
+            'action',
+            $this->url()->fromRoute('album', ['action' => 'add'])
+        );
+
+        $variables = ['form' => $this->form];
+        
+        if (! $this->getRequest()->isPost()) {
+            return $variables;
+        }
+
+        // Validation
+        $this->form->setData($this->getRequest()->getPost());
+        if (! $this->form->isValid()) {
+            return $variables;
+        }
+    
+        // …
+
+        return $this->redirect()->toRoute('album', ['action' => 'add']);
+    }
+}
+```
+
+### Create Factory for Controller
+
+Fetch the `AlbumForm` from the form element manager in a factory,
+e.g. `src/Album/Controller/AlbumControllerFactory.php`:
+
+```php
+namespace Album\Controller;
+
+use Album\Form\AlbumForm;
+use Laminas\ServiceManager\PluginManagerInterface;
+use Psr\Container\ContainerInterface;
+
+class AlbumControllerFactory
+{
+    public function __invoke(ContainerInterface $container) : AlbumController
+    {
+        /** @var PluginManagerInterface $formElementManager */
+        $formElementManager = $container->get('FormElementManager');
+        /** @var AlbumForm */ 
+        $form = $formElementManager->get(AlbumForm::class);
+
+        return new AlbumController($form);
+    }
+}
+```
+
+> ### Instantiating the Form
+>
+> The form element manager is used instead of directly instantiating the form to
+> ensure to get the input filter manager injected. This allows usage of any
+> input filter registered with the input filter managers which includes custom
+> filters and validators.  
+> Custom form elements can also be used in this way.
+>
+> Additionally the form element manager calls the `init` method _after_
+> instantiating the form, ensuring all dependencies are fully injected
+> first.
+
+## Register Form and Controller
+
+If no separate factory is required for the form, then the form element manager
+will instantiating the form class. Otherwise the form must be registered.
```

Comment:
> Otherwise the form must be registered.

A link must be added here when we have the description for the form element manager. (See #36)